Makefile

```diff
@@ -122,7 +122,10 @@ docker-push:
 	docker push ${IMG}
 
 e2e-setup:
-	bin/vela install --set installCertManager=true --image-pull-policy IfNotPresent --image-repo vela-core-test --image-tag $(GIT_COMMIT)
+	helm repo add jetstack https://charts.jetstack.io
+	helm repo update
+	helm upgrade --install --create-namespace --namespace cert-manager cert-manager jetstack/cert-manager --version v1.2.0  --set installCRDs=true --wait
```

Comment:
I think we can use the `self signed cert` version to do e2e-test, will that work?

Reply:
I tried, but one of the e2e tests failed, when creating an environment and it tries to generate a certificate, expecting an existing `Issuer`.

Reply:
OK, maybe we can refactor that in the future